engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -21,3 +21,50 @@
 
 -- PR#5668 Change the type of the 'ipsec_psk' field to allow large PSK.
 ALTER TABLE cloud.remote_access_vpn MODIFY ipsec_psk text NOT NULL;
+
+-- Management Server Status
+ALTER TABLE `cloud`.`mshost` ADD CONSTRAINT `mshost_UUID` UNIQUE KEY (`uuid`);
+CREATE TABLE `cloud`.`mshost_status` (
+  `id` bigint unsigned NOT NULL AUTO_INCREMENT COMMENT 'id',
+  `ms_id` varchar(40) DEFAULT NULL COMMENT 'the uuid of the management server record',
+  `last_start` datetime DEFAULT NULL COMMENT 'the last start time for this MS',
+  `last_stop` datetime DEFAULT NULL COMMENT 'the last stop time for this MS',
+  `last_boot` datetime DEFAULT NULL COMMENT 'the last system boot time for the host of this MS',
+  `last_down` datetime DEFAULT NULL COMMENT 'the last system shutdown time for the host of this MS',
```

Comment:
_last_system_down_ or _last_system_shutdown_

Reply:
in fact we should remove this as we cannot know for sure when a system went down. Don't you think @sureshanaparti ?